ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PurchaseButton.kt

```diff
@@ -42,6 +42,7 @@ internal fun PurchaseButton(
             IntroEligibilityStateView(
                 textWithNoIntroOffer = state.selectedLocalization.callToAction,
                 textWithIntroOffer = state.selectedLocalization.callToActionWithIntroOffer,
+                textWithMultipleIntroOffers = state.selectedLocalization.offerDetailsWithMultipleIntroOffers,
```

Comment:
Oops this needs to be 
```suggestion
                textWithMultipleIntroOffers = state.selectedLocalization.callToActionWithMultipleIntroOffers,
```

Reply:
oopsies